src/test/resources/application-integration-test.properties

```diff
@@ -14,8 +14,10 @@ test.janitorClientServiceAccountPath=rendered/client-sa-account.json
 # The credential file path to publish message to 'prod' Janitor.
 test.prodJanitorClientCredentialFilePath=rendered/tools-client-sa-account.json
 # The credential file path to create resource in test.
-test.resourceCredentialFilePath=rendered/sa-account.json
+test.resourceCredentialFilePath=rendered/cloud-access-sa-account.json
 test.prodTrackResourceProjectId=terra-kernel-k8s
 test.prodTrackResourceTopicId=crljanitor-tools-pubsub-topic
 # The project holds all created resources
 test.resourceProjectId=terra-janitor-test
+# Janitor Testing Folder
+test.parentResourceId=1074206284898
```

Comment:
Add a comment linking where this is defined and TODO (PF-67) about config rendering the value.